### ISSUE TITLE
Deploy new spanning tree topology on production switches, schedule downtime

### ISSUE THREAD

**Issue by hakasapl** [CCI-MOC/ops-issues#1188]

**Comment by joachimweyl** [comment]
Once @naved001 provides the list of affected users I will put together notifications for those targeted users as this will not affect all of our users.

**Comment by joachimweyl** [comment]
Sounds like the best approach would be to communicate with the ESI slack channel and individually with Parul Sohal as the only non-ESI member affected by this.

**Comment by joachimweyl** [comment]
@hakasapl & @msdisme please review the suggested communication below.
"Hello ESI Users,

Downtime is planned for February 5th, 2024 for parts of the MOC Alliance network. This will affect ESI and a few other switches. We will be deploying a new spanning tree topology which will require restarting many of the production switches. Please make sure all work on the ESI nodes is ready for network downtime on the 5th. We will let you know in this channel once the work is done and the network is working again.

Have a wonderful day!

Thanks!
MOC Alliance Support"

@msdisme Do we want a status page update?

**Comment by hakasapl** [comment]
This looks good to me, thanks for your help! I will send a message to OCT as well.

**Comment by msdisme** [comment]
yes please to status page updates.  Also need to let okdmoca folks know what is going on.

**Comment by joachimweyl** [comment]
@naved001 do these switches affect okdmoca?

**Comment by naved001** [comment]
@joachimweyl yes, Dereck from the OKD team is an ESI User and he's on the list I shared with you. And I think all of their hardware is now allocated via ESI.

**Comment by joachimweyl** [comment]
@msdisme then does it make sense to send only to the ESI slack group as he is in that channel?

**Comment by naved001** [comment]
@joachimweyl Dereck is not in the ESI slack channel.

He's in an ESI user so when I pulled out the **email addresses** of ESI users his email is in it and that's what I shared.

**Comment by joachimweyl** [comment]
Status update [here](https://github.com/CCI-MOC/moc-status/pull/47)

**Comment by hakasapl** [comment]
I will keep editing this comment with things that are left to another day:

* R4-PA-C21 needs a copper uplink
* Cisco switches need to be either replaced or integrated into the spanning tree
*
